deploy/roles/ultra-edit/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+- name: Uninstall Ultra Edit
+  yum: 
+    name: "{{ item }}" #UltraEdit.x86_64
+    state: removed
+    disable_gpg_check: yes
+  with_items:
+    - "{{ ue_uninstall_list }}"
```

Comment:
I think this should just be `with_items: "{{ ue_uninstall_list }}"`, although it may work either way.  This way is probably a bit cleaner.

Reply:
@tanderegg refactored, rebased, and pushed to ultraedit branch